### PR TITLE
Creates WWW.md

### PR DESCRIPTION
**Pull request by tcr**
This is intended to document our WWW infrastructure.

Right now I'm just documenting Dokku, but we need documented our Forums and tessel.io infra as well. I also intend to gut the RESOURCES.md section for web and point it here.

[Rendered](https://github.com/tessel/project/blob/70609117eb2824fe87854967abf5a9625cd04b13/WWW.md)

### CODE REVIEW

**Comment by tessel-highfive** [comment]
r? @hipsterbrown

(rust_highfive has picked a reviewer for you, use r? to override)

File: WWW.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+To dokku@107.170.4.104:NEW_APP
+   xxxxxx..xxxxxx  master -> master
+Branch master set up to track remote branch master from dokku.
+```
+
+If you want to view your app *now*, you'll have to expose the port directly on the server:
+
+```
+ufw allow <port number>
```
Comment (HipsterBrown) [inline WWW.md]: Is `ufw` a native command line tool for Mac, Linux, and Windows?

File: WWW.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+Then you can open `http://107.170.4.104:<port number>` in your browser.
```
Comment (HipsterBrown) [inline WWW.md]: That IP number can be found in the output from the `git push dokku master`, right?

File: WWW.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+2. Access to AWS via our login credentials
+
+### Setting up a new app
+
+Login to the server `ssh root@dokku.tessel.io`.
+
+To create a new app:
+
+```
+dokku apps:create NEWAPP
```
Comment (HipsterBrown) [inline WWW.md]: How does a team member get the `dokku` command line tool?

**Comment by tcr** [comment]
@HipsterBrown Re-review

**Pull request merged**